tests/integration/conftest.py

```diff
@@ -26,26 +26,34 @@
 work_dir = os.path.dirname(os.path.realpath(__file__))
 
 
+def _broker_conf():
+    broker_conf = ['transaction.state.log.replication.factor=1',
+                   'transaction.state.log.min.isr=1']
+    if TestUtils.use_group_protocol_consumer():
+        broker_conf.append('group.coordinator.rebalance.protocols=classic,consumer')
+    return broker_conf
+
+
 def create_trivup_cluster(conf={}):
     trivup_fixture_conf = {'with_sr': True,
                            'debug': True,
-                           'cp_version': '7.4.0',
-                           'version': '3.4.0',
-                           'broker_conf': ['transaction.state.log.replication.factor=1',
-                                           'transaction.state.log.min.isr=1']}
+                           'cp_version': '7.6.0',
+                           'kraft': TestUtils.use_kraft(),
+                           'version': 'trunk@f6c9feea76d01a46319b0ca602d70aa855057b07',
```

Comment:
version needs to be trunk only with new protocol, otherwise 3.7.0
```python
def _broker_version():
    return (TestUtils.use_group_protocol_consumer() and
            'trunk@f6c9feea76d01a46319b0ca602d70aa855057b07' or
            '3.7.0')
```